Tests/MongoSwiftTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -72,12 +86,14 @@ struct UnifiedTestRunner {
             // SERVER-38335.
             do {
                 let opts = RunCommandOptions(readPreference: .primary)
-                _ = try self.internalClient.anyClient.db("admin").runCommand(["killAllSessions": []], options: opts)
+                _ = try await self.internalClient.anyClient.db("admin").runCommand(
+                    ["killAllSessions": []], options: opts
```

Comment:
nit: I'm not sure why this isn't triggering a linter/formatter issue, but typically if the entire function call doesn't fit on one line, we split the arguments so they are one per line, like:
```swift
_ = try await self.internalClient.anyClient.db("admin").runCommand(
    ["killAllSessions": []],
    options: opts
)
```